packages/x-license-pro/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license-pro",
-  "version": "6.10.2",
+  "version": "6.12.0",
```

Comment:
DO you think that the `tsconfig` change warrants a release? 🤔

Reply:
No, I thought I skipped that one, I'll remove it